test/Altinn.Notifications.Sms.Tests/Sms.Core/Sending/SendingServiceTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Altinn.Notifications.Sms.Core.Configuration;
+using Altinn.Notifications.Sms.Core.Dependencies;
+using Altinn.Notifications.Sms.Core.Sending;
+using Altinn.Notifications.Sms.Core.Status;
+using Moq;
+
+namespace Altinn.Notifications.Sms.Tests.Sms.Core.Sending;
+
+public class SendingServiceTests
+{
+    private readonly TopicSettings _topicSettings;
+
+    public SendingServiceTests()
+    {
+        _topicSettings = new()
+        {
+            SmsStatusUpdatedTopicName = "SmsStatusUpdatedTopicName"
+        };
+    }
+
+    [Fact]
+    public async Task SendAsync_GatewayReferenceGenerated_PublishedToExpectedKafkaTopic()
```

Comment:
since it always published to the same topic in these cases, should we rather focus on the contents published? Gatewayref vs sendstatus?